app/assets/javascripts/application.js

```diff
@@ -6,7 +6,7 @@
 //= require govuk_publishing_components/components/radio
 //= require govuk_publishing_components/components/step-by-step-nav
 //= require helpers
-//= require components/escape-link
+//= require components/quick-escape-banner
```

Comment:
Thanks for pointing this out @kevindew - I went with it because of what it is known as outside of GDS https://health.columbia.edu/content/quick-escape-button

Happy to rename - makes sense what you're saying